sessionctx/binloginfo/binloginfo.go

```diff
@@ -189,8 +185,7 @@ func MockPumpsClient(client binlog.PumpClient) *pumpcli.PumpsClient {
 		ClusterID:          1,
 		Pumps:              pumpInfos,
 		Selector:           pumpcli.NewSelector(pumpcli.Range),
-		RetryTime:          1,
-		BinlogWriteTimeout: 15 * time.Second,
+		BinlogWriteTimeout: time.Second,
```

Comment:
Change it?

Reply:
we change the retry strategy in pump client, must retry longer than BinlogWriteTimeout, set 1 second here to make test faster